website/docs/docs/core/connect-data-platform/teradata-setup.md

```diff
@@ -124,6 +157,91 @@ For the full description of the connection parameters see https://github.com/Ter
+| Cross-database macros | hash                          | :white_check_mark:    | custom macro provided, see [compatibility note](#hash)                 |
+| Cross-database macros | replace                       | :white_check_mark:    | custom macro provided                                                  |
+| Cross-database macros | type_string                   | :white_check_mark:    | custom macro provided                                                  |
+| Cross-database macros | last_day                      | :white_check_mark:    | no customization needed, see [compatibility note](#last_day)           |
+| Cross-database macros | width_bucket                  | :white_check_mark:    | no customization
+
+
+#### examples for cross DB macros
+  ##### <a name="replace"></a>replace
+  {{ dbt.replace("string_text_column", "old_chars", "new_chars") }}
+  {{ replace('abcgef', 'g', 'd') }}
+
+  ##### <a name="date_trunc"></a>date_trunc
+  {{ dbt.date_trunc("date_part", "date") }}
+  {{ dbt.date_trunc("DD", "'2018-01-05 12:00:00'") }}
+
+  ##### <a name="datediff"></a>datediff
+  `datediff` macro in teradata supports difference between dates. Differece between timestamps is not supported.
+
+  ##### <a name="hash"></a>hash
+
+  `Hash` macro needs an `md5` function implementation. Teradata doesn't support `md5` natively. You need to install a User Defined Function (UDF):
+  1. Download the md5 UDF implementation from Teradata (registration required): https://downloads.teradata.com/download/extensibility/md5-message-digest-udf.
+  1. Unzip the package and go to `src` directory.
+  1. Start up `bteq` and connect to your database.
+  1. Create database `GLOBAL_FUNCTIONS` that will host the UDF. You can't change the database name as it's hardcoded in the macro:
+      ```sql
+      CREATE DATABASE GLOBAL_FUNCTIONS AS PERMANENT = 60e6, SPOOL = 120e6;
+      ```
+  1. Create the UDF. Replace `<CURRENT_USER>` with your current database user:
+      ```sql
+      GRANT CREATE FUNCTION ON GLOBAL_FUNCTIONS TO <CURRENT_USER>;
+      DATABASE GLOBAL_FUNCTIONS;
+      .run file = hash_md5.btq
+      ```
+  1. Grant permissions to run the UDF with grant option.
+      ```sql
+      GRANT EXECUTE FUNCTION ON GLOBAL_FUNCTIONS TO PUBLIC WITH GRANT OPTION;
+      ```
+  ##### <a name="last_day"></a>last_day
+
+  `last_day` in `teradata_utils`, unlike the corresponding macro in `dbt_utils`, doesn't support `quarter` datepart.
+
+## Limitations
+
+### Transaction mode
+Only ANSI transaction mode is supported.
+
+## Credits
+
+The adapter was originally created by [Doug Beatty](https://github.com/dbeatty10). Teradata took over the adapter in January 2022. We are grateful to Doug for founding the project and accelerating the integration of dbt + Teradata.
+
+## License
+
+The adapter is published using Apache-2.0 License. Please see the link (ADD LINK HERE) for terms and conditions, such as creating derivative work and the support model. 
```

Comment:
hey @tallamohan , thanks for opening this up! the deployment was failing because it expected a `LICENSE` link. Instead, I've add a 'add link here' callout for you to add the correct link.